include/libtorrent/aux_/session_udp_sockets.hpp

```diff
@@ -56,6 +59,42 @@ namespace libtorrent { namespace aux {
 		bool write_blocked = false;
 	};
 
+	struct outgoing_udp_socket final : session_udp_socket
+	{
+		explicit outgoing_udp_socket(io_service& ios, std::string const& dev, bool ssl_)
```

Comment:
I don't think you need "explicit" here, since the constructor takes more than one argument